wallet-new/server/Cardano/Wallet/API/V1/Swagger.hs

```diff
@@ -503,6 +583,404 @@ Disable TLS (Not Recommended)
 -----------------------------
 
 If needed, you can disable TLS by providing the `--no-tls` flag to the wallet or by running a wallet in debug mode with `--wallet-debug` turned on.
+
+
+Common Use-Cases
+================
+
+Sending Money to Multiple Recipients
+------------------------------------
+
+As seen in [Sending ADA](#section/Getting-Started/Sending-ADA), you can send _ADA_ to
+another party using the [`POST /api/v1/transactions`](#tag/Transactions%2Fpaths%2F~1api~1v1~1transactions%2Fpost) endpoint.
+Important to notice is the type of the field `destinations`: it's a list, enabling you to provide more
+than one destination. Each destination is composed of:
+
+- An address
+- A corresponding amount
+
+The overall transaction corresponds to the sum of each outputs. For instance, to send money to
+two parties simultaneously:
+
+```
+curl -X POST https://localhost:8090/api/v1/transactions \
+  -H "Accept: application/json; charset=utf-8" \
+  -H "Content-Type: application/json; charset=utf-8" \
+  --cacert ./scripts/tls-files/ca.crt \
+  -d '{
+  "destinations": [
+    {
+      "amount": 14,
+      "address": "A7k5bz1QR2...Tx561NNmfF"
+    },
+    {
+      "amount": 42,
+      "address": "B56n78WKE8...jXAa34NUFz"
+    }
+  ],
+  "source": {
+    "accountIndex": 0,
+    "walletId": "Ae2tdPwUPE...8V3AVTnqGZ"
+  },
+  "spendingPassword": "5416b2988745725998907addf4613c9b0764f04959030e1b81c603b920a115d0"
+}'
+```
+
+
+Estimating Transaction Fees
+---------------------------
+
+When you submit a transaction to the network, some fees apply depending on, but not only, the
+selected grouping policy and the available inputs on the source wallet. There's actually a
+trade-off between fees, cryptographic security, throughput and privacy. The more inputs are
```

Comment:
Do we have a doc or link pointing to an explanation of the trade-offs? Or is this something we'd expect someone reading these docs to know :smile: